draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -108,58 +108,96 @@ contributor:
     fullname: Gyan Mishra
     org: Verizon
     email: gyan.s.mishra@verizon.com
+  
+  -
+    ins: S. Melin
+    fullname: Stefan Melin
+    org: Telia
+    email: stefan.melin@teliacompany.com
+
+  -
+    ins: M. Hossein Poor
+    fullname: Majid Hossein Poor
+    org: Telstra
+    email: majid.hosseinpoor@team.telstra.com
+
+  -
+    ins: D. Demeter
+    fullname: Dacian Demeter
+    org: Telus
+    email: dacian.demeter@telus.com 
 
 normative:
   OIF-CMIS:
     title: "OIF Implementation Agreement (IA) Common Management Interface Specification (CMIS))"
     author:
-      org: OIF Forum
+    org: OIF Forum
     date: 27 April 2022
     seriesinfo: OIF CMIS IA
     target: https://www.oiforum.com/wp-content/uploads/OIF-CMIS-05.2.pdf
```

Comment:
Can be updated in next revision but CMIS 5.3 has been published.

Reply:
Done